core/amber/src/main/python/core/architecture/sendsemantics/broad_cast_partitioner.py

```diff
@@ -19,7 +19,7 @@ def __init__(self, partitioning: BroadcastPartitioning):
         super().__init__(set_one_of(Partitioning, partitioning))
         self.batch_size = partitioning.batch_size
         self.batch: list[Tuple] = list()
-        self.receivers: List[ActorVirtualIdentity] = partitioning.receivers
+        self.receivers: List[ActorVirtualIdentity] = partitioning.channels
```

Comment:
the type is no longer `List[ActorVirtualIdentity]` right?

Reply:
Fixed.